email-notification-service/Dockerfile

```diff
@@ -1,13 +1,21 @@
-FROM node:alpine
+FROM node:18.4.0-alpine
 
 WORKDIR /home/node
 
-COPY package*.json ./
+COPY package.json ./
+COPY package-lock.json ./
 RUN npm ci
 
 COPY src src/
 COPY tsconfig.json .
 
 RUN npm run build
 
+# Create user
+RUN adduser -S -D -h /home/node newuser
```

Comment:
Give the user a meaningful name

Reply:
suggest one please. Newuser is a meaningful name for this purpose